pkg/rehearse/jobs.go

```diff
@@ -303,7 +360,18 @@ func (jc *JobConfigurer) ConfigurePeriodicRehearsals(periodics config.Periodics)
 	filteredPeriodics := filterPeriodics(periodics, jc.loggers.Job)
 	for _, job := range filteredPeriodics {
 		jobLogger := jc.loggers.Job.WithField("target-job", job.Name)
-		if err := jc.configureJobSpec(job.Spec, jc.loggers.Debug.WithField("name", job.Name)); err != nil {
+		variant := ""
+		if newVariant, ok := job.Labels[jobconfig.ProwJobLabelVariant]; ok {
+			variant = newVariant
+		}
+		var orgrepo, branch string
+		if len(job.ExtraRefs) != 0 {
```

Comment:
This is *almost* an `Info{}` -- should be a smell that passing these around together but not in a structure is not the most appropriate ;)